cupy/testing/array.py

```diff
@@ -7,7 +7,7 @@
 
 def assert_allclose(actual, desired, rtol=1e-7, atol=0, err_msg='',
                     verbose=True):
-    """Raises an AssertionError if objects are not equal up to desired tolerance.
+    """Raises an AssertionError if objects are not equal up to desired tolerance.  # NOQA
```

Comment:
`# NOQA` needs to be placed at the end of the block comment. Otherwise it is rendered to docs. ditto for others.